packages/js/src/ui/components/Header/Header.tsx

```diff
@@ -1,13 +1,12 @@
-import { useAppearance } from 'src/ui/context';
-import { cn, useStyle } from 'src/ui/helpers';
+import { cn, useStyle } from '../../helpers';
 import { ActionsContainer } from './ActionsContainer';
 import { StatusDropdown } from './StatusDropdown';
 
 export const Header = () => {
   const style = useStyle();
 
   return (
-    <div class={style('inboxHeader', cn('nt-flex nt-justify-between nt-items-center nt-self-stretch nt-py-5 nt-px-6'))}>
+    <div class={style('inboxHeader', cn('nt-flex nt-justify-between nt-items-center nt-w-full'))}>
```

Comment:
```suggestion
    <div class={style('inboxHeader', cn('nt-flex nt-justify-between nt-items-center nt-w-full  nt-py-5 nt-px-6'))}>
```
Add back the padding